src/lambda/commands/uploadLambda.ts

```diff
@@ -212,9 +212,16 @@ export class UploadLambdaWizard extends Wizard<UploadLambdaWizardState> {
         super({ initState: { lambda } })
         this.form.lambda.region.bindPrompter(() => createRegionPrompter().transform(region => region.id))
 
-        if (invokePath && fs.statSync(invokePath.fsPath).isDirectory()) {
+        if (
+            (invokePath && fs.statSync(invokePath.fsPath).isDirectory()) ||
+            invokePath?.fsPath.endsWith('template.yaml')
```

Comment:
1. This assumes that we won't support other filenames besides template.yaml in the future
2. the endsWith(...) check is duplicated below. Could be stored in a `const isFile = ...` above.
3. Why not check "file exists" instead of checking for "template.yaml" ?